src/vs/workbench/electron-browser/commands.ts

```diff
@@ -423,3 +424,16 @@ export function registerCommands(): void {
 		});
 	});
 }
+
+namespace GlobalQuickOpenCommand {
+	export const ID = 'vscode.quickOpen';
```

Comment:
In a previous comment, @jrieken commented on the exact line that i've modified in the action to call the command instead of the service.
> These UI gestures basically all call executeCommand with their assigned id. That is why running exec-command from within an action is a double loop (the run is already the result of calling exec-command)

I'll try using the same ID and see if it works, but it could just execute both the command and the action, or maybe have a conflict between them, or randomly execute either the action or the command. I'll investigate this and come back with an actual response instead of an assumption.